pages/cql/bcs-logic-2018.cql

```diff
@@ -64,72 +91,120 @@ define "Is Age 52 to 74 at End":
 	AgeInYearsAt(end of "Measurement Period") between 52 and 74
 
 define "Is In Hospice":
```

Comment:
This looks to have been simplified from the version in the Hospice-2.0.000.cql... Is it accounting for all cases where a patient is in Hospice care?

Reply:
The HEDIS spec "General Guideline 20: Members in Hospice" explicitly says to use the "Hospice Value Set" to identify patients in Hospice.  Also, the FHIR model is different than the QDM model that Hospice-2.0.000.cql is written for.

Reply:
The reason it was written that way is that it is based on hospital data, so it is looking for inpatient encounters that resulted in a discharge to hospice care. I agree that if we had all encounter data for the patient, the test you've written would work, but in the case where all we have is hospital data, we have to look for a discharge disposition that indicates hospice care.

Another aspect the other logic is looking for is orders or performance of procedures related to hospice care. So I think a better approach for now is to just translate what was expressed in the QDM to the FHIR analog.